Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss

```diff
@@ -297,93 +293,67 @@ $snackbar-padding-y-multi: 1.25rem !default;
 .snackbar-stack-top-end .snackbar.snackbar-show {
     transition: all var(--stack-transition-duration, 1000);
     animation: showTop var(--stack-animation-duration, 500) forwards;
-    opacity: 1;
 }
 
 .snackbar-stack-top .snackbar-hide,
 .snackbar-stack-top-start .snackbar-hide,
 .snackbar-stack-top-end .snackbar-hide {
     transition: all var(--stack-transition-duration, 1000);
-    animation: hideTop var(--stack-animation-duration, 500) forwards;
-    opacity: 0;
+    animation: hideTop var(--stack-animation-duration, 500) backwards;
 }
 
 .snackbar-stack-bottom .snackbar.snackbar-show,
 .snackbar-stack-bottom-start .snackbar.snackbar-show,
 .snackbar-stack-bottom-end .snackbar.snackbar-show {
     transition: all var(--stack-transition-duration, 1000);
     animation: showBottom var(--stack-animation-duration, 500) forwards;
-    opacity: 1;
 }
 
 .snackbar-stack-bottom .snackbar-hide,
 .snackbar-stack-bottom-start .snackbar-hide,
 .snackbar-stack-bottom-end .snackbar-hide {
     transition: all var(--stack-transition-duration, 1000);
-    animation: hideBottom var(--stack-animation-duration, 500) forwards;
-    opacity: 0;
+    animation: hideBottom var(--stack-animation-duration, 500) backwards;
 }
 
 @keyframes showTop {
     0% {
-        max-height: 0;
-        padding: 0;
-        transform: translate(-50%) translateY(-50px);
         opacity: 0;
     }
 
     100% {
-        max-height: 150px;
-        padding: $spacer-sm $spacer;
```

Comment:
Do we need to remove these?

Reply:
I already removed that in last commit, do you see it?